### PR TITLE
Add marine equipment specialist document

### PR DESCRIPTION
**Pull request by minhngocd**
New specialist publisher for Marine Equipment Approved Recommendations by Maritime Coastguard Agency.

### CODE REVIEW

File: lib/documents/schemas/marine_equipment_approved_recommendations.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      "key": "category",
+      "name": "Category",
+      "short_name": "Recommendation type",
+      "type": "text",
+      "preposition": "With recommendation type",
+      "display_as_result_metadata": true,
+      "filterable": true,
+      "allowed_values": [
+        {
+          "label": "Life-saving appliances",
```
Comment (baisa) [inline lib/documents/schemas/marine_equipment_approved_recommendations.json:101]: Should we have a dash in `Life-saving`?

File: lib/documents/schemas/marine_equipment_approved_recommendations.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        {
+          "label": "Fire protection equipment",
+          "value": "fire-protection-equipment"
+        },
+        {
+          "label": "Navigation equipment",
+          "value": "navigation-equipment"
+        },
+        {
+          "label": "Radio-communication equipment",
```
Comment (baisa) [inline lib/documents/schemas/marine_equipment_approved_recommendations.json:117]: Similarly here

**Review by baisa — COMMENTED** [review]
This is looking good! Just added minor comments. We could consider commits cleanup - as removing validation isn’t separate logic bit, we could probably include in in the first commit. Pre production flag commits could be also merged into one.

File: lib/documents/schemas/marine_equipment_approved_recommendations.json (hunk 1 above)
Comment (minhngocd) [reply lib/documents/schemas/marine_equipment_approved_recommendations.json:101]: This came from their requirements documentation - we can double check with MCA when we run a test in integration?

File: lib/documents/schemas/marine_equipment_approved_recommendations.json (hunk 2 above)
Comment (minhngocd) [reply lib/documents/schemas/marine_equipment_approved_recommendations.json:117]: Same, it came from their requirements documentation - we can double check with MCA when we run a test in integration?

**Review by baisa — APPROVED** [review]

**Review by baisa — APPROVED** [review]

**Comment by baisa** [comment]
LGTM, great work 😊

**Pull request merged**